src/astro/databases/snowflake.py

```diff
@@ -146,6 +157,22 @@ class SnowflakeDatabase(BaseDatabase):
     logic in other parts of our code-base.
     """
 
+    NATIVE_LOAD_EXCEPTIONS: Any = (
+        ValueError,
```

Comment:
@sunank200 Why we have ValueError and AttributeError in this list?

Reply:
This is to catch ValueError and ArributeError for `run_sql()` method of sqlalchemy. This can be improved by raising these two errors only while calling `run_sql()`

Reply:
As per discussion, we should be handling the generic exception at the source and raise a custom exception that can be handled in via `NATIVE_LOAD_EXCEPTIONS`.

Reply:
Adding separate issues for this and tracking this in separate PR. But this should not be a blocker the release